docker/Dockerfile_stack

```diff
@@ -5,9 +5,11 @@ ARG NPROC=4
 RUN apt-get update && apt-get install -y \
         build-essential \
         gfortran \
-        wget \
-        git \
+        libopenmpi-dev \
```

Comment:
I tried mpich and moab would not build.  It was easier to just patch mpiexec than investigate it.